test/e2e/config/azure-dev.yaml

```diff
@@ -155,8 +155,6 @@ providers:
         targetName: "cluster-template-private.yaml"
       - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-ci-version.yaml"
         targetName: "cluster-template-conformance-ci-artifacts.yaml"
-      - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-ci-version-windows.yaml"
-        targetName: "cluster-template-conformance-ci-artifacts-windows.yaml"
       - sourcePath: "${PWD}/templates/test/ci/cluster-template-prow-ci-version-windows-containerd-2022.yaml"
         targetName: "cluster-template-conformance-ci-artifacts-windows-containerd-2022.yaml"
```

Comment:
can we also rename `cluster-template-prow-ci-version-windows-containerd-2022` to `cluster-template-prow-ci-version-windows-2022` ?

Reply:
I think this will rely on change the logic in the e2e_conformance: https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2315

Lets follow up with a separate pr for that?